src/pages/Bridge/BridgeOverview/components/IssueForm/IssueForm.tsx

```diff
@@ -169,10 +201,21 @@ const IssueForm = ({ requestLimits, dustValue, issueFee }: IssueFormProps): JSX.
 
   const isSelectingVault = form.values[BRIDGE_ISSUE_CUSTOM_VAULT_SWITCH];
 
-  const isBtnDisabled = isFormDisabled(form);
+  const isBtnDisabled = isTransactionFormDisabled(form, transaction.fee);
+
+  const griefingCollateralCurrencyBalance =
+    griefingCollateralTicker === undefined ? undefined : getBalance(griefingCollateralTicker)?.free;
```

Comment:
All of the. checks you are doing for undefined are uncessary, where a simple `!griefingCollateralTicker` negation does the job. Having all these undefined checks makes the code bigger.